lib/ui/views/ib/ib_page_view.dart

```diff
@@ -397,36 +399,45 @@ class _IbPageViewState extends State<IbPageView> {
       }
 
       buttons.add(
-        AnimatedOpacity(
-          duration: const Duration(milliseconds: 500),
-          opacity: _isFabsVisible ? 1.0 : 0.0,
-          child: FloatingActionButton(
-            heroTag: 'previousPage',
-            mini: true,
-            backgroundColor: Theme.of(context).primaryIconTheme.color,
-            onPressed: () {
-              //If FAB are not visible do not do anything.
-              if (!_isFabsVisible) {
-                return;
-              }
-              widget.setPage(widget.chapter.prev);
-            },
-            child: Showcase(
-              key: _model.prevPage,
-              description: 'Tap to navigate to previous page',
-              targetPadding: const EdgeInsets.all(12.0),
-              targetShapeBorder: const CircleBorder(),
-              onTargetClick: () {
-                widget.setShowCase(widget.showCase.copyWith(prevButton: true));
-                widget.setPage(widget.chapter.prev);
-              },
-              disposeOnTap: true,
-              child: const Icon(
-                Icons.arrow_back_rounded,
-                color: IbTheme.primaryColor,
-              ),
-            ),
-          ),
+        ChangeNotifierProvider<IbFloatingButtonState>.value(
+          value: _ibFloatingButtonState,
+          child: Builder(builder: (context) {

```

Comment:
@Tanmoy741127 remove this unnecessary builder widget.

Reply:
Okk